CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ option(USE_CUDNN "Use CUDNN library" OFF)
 option(USE_DOXYGEN "Build documentation with Doxygen" ON)
 option(USE_FBGEMM "Use FBGEMM" OFF)
 option(USE_MKL "Compile with MKL support" ON)
+option(USE_DNNL "Compile with oneDNN support" OFF)
```

Comment:
Ok, I think for the oneDNN branch this should be a default.

Reply:
It's left from upstream behaviour. It can be changed before merge

Reply:
yeah i tried to build without oneDNN but oneDNN is implicit in this branch